plugins/modules/sonic_route_maps.py

```diff
@@ -0,0 +1,1608 @@
+#  match interface Ethernet16
+#  set as-path prepend 200,300,400
+#  set metric 8000
+#  set ipv6 next-hop global 45::90
+# !
+# route-map rm3 deny 285
+#  match evpn route-type macip
+#  match local-preference 14783
+#  call rm1
+#  match peer 87:95:15::53
+#  match source-protocol bgp
+#  match origin igp
+#  set community no-advertise
+#  set metric rtt
+#  set origin igp
+# !
+# route-map rm4 permit 480
+#  match evpn route-type prefix
+#  match source-protocol static
+#  set metric -rtt
+
+
+# Using "overridden" state to override all existing configuration with new
+# configuration
+#
+# Before state:
+# ------------
+#
+# sonic# show running-configuration route-map
+# !
+# route-map rm1 permit 80
+#  match as-path bgp_as2
+#  match evpn route-type multicast
+#  match ip address prefix-list ip_pfx_list1
+#  match ipv6 address prefix-list ipv6_pfx_list1
+#  match interface Vlan7
+#  match community bgp_comm_list3
+#  match ext-community bgp_ext_comm1
+#  match tag 7284
+#  match local-preference 8000
+#  match source-vrf Vrf1
+#  match peer PortChannel 14
+#  match source-protocol bgp
+#  match metric 400
+#  match origin egp
+#  set as-path prepend 188,257
+#  set community no-advertise
+#  set extcommunity rt 30:40
+#  set extcommunity rt 20:20
+#  set comm-list bgp_comm_list2 delete
+#  set metric +rtt
+#  set ip next-hop 10.48.16.18
+#  set ipv6 next-hop global 30::30
+#  set local-preference 635
+#  set origin egp
+#  set weight 93471
+# !
+# route-map rm1 deny 3047
+#  match as-path bgp_as3
+#  match evpn route-type multicast
+#  match ext-community bgp_ext_comm2
+#  match peer Ethernet6
+#  match source-protocol ospf
+#  match origin igp
+#  set metric -rtt
+#  set origin incomplete
+# !
+# route-map rm2 permit 100
+#  match interface Ethernet16
+#  set as-path prepend 200,300,400
+#  set metric 8000
+#  set ipv6 next-hop global 45::90
+# !
+# route-map rm3 deny 285
+#  match evpn route-type macip
+#  match local-preference 14783
+#  call rm1
+#  match peer 87:95:15::53
+#  match source-protocol bgp
+#  match origin igp
+#  set community no-advertise
+#  set metric rtt
+#  set origin igp
+# !
+# route-map rm4 permit 480
+#  match evpn route-type prefix
+#  match source-protocol static
+#  set metric -rtt
+# ------------
+- name: Override all route map configuration with new configuration
+  dellemc.enterprise_sonic.sonic_route_maps:
+     config:
+       - map_name: rm5
+         action: permit
+         sequence_num: 250
+         match:
+           interface: Ethernet28
+         set:
+           as_path_prepend: 150,275
+           metric: 7249
+     state: overridden
+
+# After state:
+# ------------
+#
+# sonic# show running-configuration route-map
+# !
+# route-map rm5 permit 250
+#  match interface Ethernet28
+#  set as-path prepend 150,275
+#  set metric 7249
+
+
+# Using "overridden" state to override all existing configuration with new
+# configuration. (Restore the "after" state from "merged_example_02.txt".)
```

Comment:
Reference to 'merged_example_02.txt' is not valid in module Documentation and can be removed.

Reply:
I have removed this, but, so I know for the future, why was specifying the merged example reference not valid?